### ISSUE TITLE
Incompatibility with Really Simple History library

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#40]
```
Please try the attached script on IE6. It is a blank page that just
includes IE7 and the Really Simple History library
(http://reallysimplehistory.googlecode.com/)

Any time I click (anywhere), I get an error: '0' is null or not an object
(line 1339)

UserAgent: Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1; InfoPath.1;
.NET CLR 2.0.50727; .NET CLR 3.0.04506.648; .NET CLR 3.5.21022)

I am not sure whose code needs to be revised (IE7 or RSH), but would really
appreciate if you could take a look. Thanks!
```

Original issue reported on code.google.com by `root.n...@gmail.com` on 22 Feb 2008 at 3:18

Attachments:
- [ie7-rsh.html](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-40/comment-0/ie7-rsh.html)

**Comment by GoogleCodeExporter** [comment]
```
I'm using IE8 and rsh and everything works fine.
```

Original comment by `roy...@gmail.com` on 7 Sep 2008 at 11:21

**Comment by GoogleCodeExporter** [comment]
```
I can't recreate this problem. It was probably fixed in 2.0 beta3.
```

Original comment by `dean.edw...@gmail.com` on 11 Feb 2010 at 6:02
- Changed state: **Fixed**
